nextroute/schema/output.go

```diff
@@ -6,60 +6,96 @@ import (
 	"github.com/nextmv-io/sdk/nextroute/check"
 )
 
-// SolutionOutput represents a solutions as JSON.
+// SolutionOutput represents a solution as JSON.
 type SolutionOutput struct {
-	Unplanned []StopOutput    `json:"unplanned"`
-	Vehicles  []VehicleOutput `json:"vehicles"`
+	// Unplanned is the list of stops that were not planned in the solution.
+	Unplanned []StopOutput `json:"unplanned"`
+	// Vehicles is the list of vehicles that were used in the solution.
```

Comment:
Doesn't this list _all_ vehicles of the input? (not only the ones _used_ in the solution - this makes it sound related to the activation penalty)